File: src/sdk-reference/java/1/collection/validate-specifications/snippets/validate-specifications.java
try {
    String specifications = "{ \"strict\": false, \"fields\": { \"license\": { \"mandatory\": true, \"type\": \"string\" } } }";
    ValidationResponse validationResponse = kuzzle.getCollection().validateSpecifications("nyc-open-data", "yellow-taxi", specifications);

    if (validationResponse.getValid()) {
      System.out.println("Success");
    }
} catch (KuzzleException e) {
    System.err.println(e.getMessage());
}

File: src/sdk-reference/go/1/collection/validate-specifications/snippets/validate-specifications.go
specifications := json.RawMessage(`{ "nyc-open-data": { "yellow-taxi": { "strict": false, "fields": { "license": { "mandatory": true, "type": "string" } } } } }`)
vr, err := kuzzle.Collection.ValidateSpecifications(specifications, nil)

if err != nil {
  log.Fatal(err)
} else if vr.Valid == true {
  fmt.Println("Success")
}
